test/e2e/egress_test.go

```diff
@@ -552,7 +717,7 @@ func (data *TestData) createEgress(t *testing.T, generateName string, matchExpre
 }
 
 func (data *TestData) waitForEgressRealized(egress *v1alpha2.Egress) (*v1alpha2.Egress, error) {
-	err := wait.PollImmediate(200*time.Millisecond, 3*time.Second, func() (done bool, err error) {
+	err := wait.PollImmediate(200*time.Millisecond, 5*time.Second, func() (done bool, err error) {
```

Comment:
So 3 seconds is enough in some cases? I feel it may indicate there is some problem but I don't think it's related to this PR so I'm fine with the change. We'd better to look at the logs to understand why the delay is so long.

Reply:
3 seconds is enough in most cases. I have tested hundreds of times in my local env (dual-stack cluster) and nerver failed.